Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with prometheus-operator.
+
+## Two approaches to utilizing ServiceMonitors
```

Comment:
No the beginning is fine, I was talking about this

```markdown
## Two approaches to utilizing ServiceMonitors
What approach is the "best" is currently mostly a "personal" preference.
```

We need to give users guidance, things like that make users uneasy. It's better to go one way and if necessary rethink the way we are showing.